x-pack/plugins/security_solution/common/api/entity_analytics/asset_criticality/common.schema.yaml

```diff
@@ -18,62 +18,69 @@ components:
       in: query
       required: true
       schema:
-        $ref: '#/components/schemas/IdField'
-      example: 'host.name'
+        $ref: "#/components/schemas/IdField"
+      example: "host.name"
       description: The field representing the ID.
 
   schemas:
     IdField:
       type: string
       enum:
-        - 'host.name'
-        - 'user.name'
+        - "host.name"
```

Comment:
Existing specs use single quotes. Is it possible to change auto-formatting configuration to format with single quotes?

Reply:
I am using VS code editor default `yaml` formatter. We should add these files to prettier and let it handle formating for us.

Reply:
Could you describe your actions to format `yaml` files? Have you installed any plugins?

Reply:
I have the YAML extension by Red Hat installed, and it autoformats on save.

```
    "editor.formatOnSave": true,
    ...
    "[yaml]": {
        "editor.defaultFormatter": "redhat.vscode-yaml"
    },
   ```